README.rst

```diff
@@ -95,10 +95,13 @@ two test files cannot be named the same, even if they are in different sub-direc
 Code coverage is provided by the `pytest-cov <https://pytest-cov.readthedocs.io/en/latest/>`_
```

Comment:
Somewhere in the README or where you think appropriate, could you like to the sources that explain why using `src/` is beneficial (for example the `pytest` best practices etc). I'd like to have all of the supporting links available for others to read.